### PR TITLE
feat: added side menu/ contacts/ settings/ data

### PR DESCRIPTION
**Pull request by KarachunVladislav01**
Ready to review.

### CODE REVIEW

File: sources/views/data.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+		};
+		const multiView = {
+			cells: [
+				{
+					id: "countries",
+					rows: [new DataTable(this.app, "", countries)]
+				},
+				{
+					id: "statuses",
+					rows: [new DataTable(this.app, "", statuses)]
```
Comment (origamirin) [inline sources/views/data.js]: Use localId insted of regular id

File: sources/views/contacts.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+			view: "list",
+			localId: "userList",
+			select: true,
+			scroll: false,
+			template:
+				"<div class= 'users-list--flex'>{common.userInfo()}{common.deleteIcon}</div>",
+			type: {
+				userInfo: function(obj) {
+					return `<span>${obj.Name}, ${obj.Email}</span>`;
+				},
```
Comment (origamirin) [inline sources/views/contacts.js]: You can use arrow function and template strings here

File: sources/styles/app.css [hunk 3]
```diff
@@ -0,0 +1,69 @@
+.webixappstart {
```
Comment (Zwillinge) [inline sources/styles/app.css]: please clean css and leave only rules you need

File: sources/views/settings.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
+import { JetView } from "webix-jet";
+
+export default class Settings extends JetView {
+	config() {
+		const changeLanguageButtons = {
+			view: "segmented",
+			value: "EN",
+			options: [
+				{
+					localId: "en",
```
Comment (Zwillinge) [inline sources/views/settings.js]: localIds work with Webix views only. 
data item, control options - use regular ids, otherwide they will not work (generate auto ids in backgroud and we will not be able to work with this menu/control )

Comment (KarachunVladislav01) [reply sources/views/settings.js]: Ready to review.

File: sources/views/data.js (hunk 1 above)
Comment (Zwillinge) [reply sources/views/data.js]: + 1

provide localIds for Webix views and normal ids for segmented options

Comment (KarachunVladislav01) [reply sources/views/data.js]: Ready to review.

**Pull request merged**